sdk/core/core/README.md

```diff
@@ -1,10 +1,16 @@
-# Azure SDK Core Library
+# Azure SDK Core Library for C
 
-Azure Core library (`az_core`) provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C programming language. These libraries follow the Azure SDK Design Guidelines for Embedded C.
+Azure Core library for C (`az_core`) provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C programming language. These libraries follow the Azure SDK Design Guidelines for Embedded C.
```

Comment:
Do we use the 'az_core' anywhere else? It is not in the code base at all. Perhaps remove this parenthetical?

Reply:
In CMake files we use it to reference the package.  Its relevant for those building packages on top of core